src/main/java/org/broadinstitute/consent/http/service/EmailService.java

```diff
@@ -233,6 +241,71 @@ private void sendNewDARRequestEmail(
         user.getUserId());
   }
 
+  public void sendExpirationNotices() {
+    sendDARExpirationReminderNotices();
+    sendDARExpirationNotices();
+  }
+
+  private void sendDARExpirationNotices() {
```

Comment:
Instead of adding this to EmailService, I've been thinking that these should really be in their specific service classes. In this case, I would put the methods in `DataAccessRequestService`

Reply:
I think we might want to do this on a holistic level.  These aren't the only methods like this in the class like this.  I'd support a ticket to do that kind of refactor.